extern crate redict;
use redict::BoxError;

fn main() -> std::result::Result<(), BoxError> {
    Ok(())
}
